src/fits.cc

```diff
@@ -637,10 +667,25 @@ void Fits::forEachKey(HeaderIterationFunctor &functor) {
 
 namespace {
 
-bool isKeyIgnored(std::string const &key) {
-    return key == "SIMPLE" || key == "BITPIX" || key == "EXTEND" || key == "GCOUNT" || key == "PCOUNT" ||
-           key == "XTENSION" || key == "TFIELDS" || key == "BSCALE" || key == "BZERO" ||
-           key.compare(0, 5, "NAXIS") == 0;
+bool isKeyIgnored(std::string const &key, bool write=false) {
+    for (auto const & kk : ignoreKeys) {
```

Comment:
I'd feel a little happier if this was a O(1) hash lookup instead of an O(n) list iteration because this is going to be called for every key in every HDU.  I understand that would be harder for the `Starts` items, although you could hash the first 4 or 5 letters.